apps/charterafrica/src/lib/ecosystem/utils.js

```diff
@@ -0,0 +1,21 @@
+function localize(data, locales = ["en"]) {
+  const getValue = (value, locale) => {
+    if (value?.[locale] !== undefined) {
+      return value?.[locale];
+    }
+    return value;
```

Comment:
```suggestion
    return value?.[locale] ?? value;
```

Reply:
@kilemensi  I am iffy about this.

```js
{
 theme: { pt: null }
}
```

will result in something like

```js
{
 pt: {  theme: { pt: null } }
}
```

Reply:
Yes @koechkevin, `??` only handles `undefined`. If you want to handle `undefined` _and_ `null`, repalce it wil `||`